crates/sui-graphql-rpc/schema.graphql

```diff
@@ -0,0 +1,840 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Open Questions
+# --------------
+# - Rules for nullabilit/non-nullability
+# - Pattern for multi-gets
```

Comment:
What's the best way to encode querying for multiple objects, transactions, etc?

Reply:
personally I like having filter objects with operations as it provides good composability/extensibility, the downside is it can be hard to implement well on the backend, and will not work well if you have constraints on what kinds of filters can be combined. 

```graphql
query {
  objects(
    filter: {
      id: { in: [id1, id2] }
    }
  ) {
    id
  }
}
```

Reply:
I think this comment would be addressed by merging `queryObjects` with `objects`.